### DISCUSSION TITLE
MACOSX Apple M1 chip

### DISCUSSION THREAD

**Discussion by cohen-sydney** [AlexandreRouma/SDRPlusPlus#1180] [General]
is there a package for the Apple M chip rather than the Apple Intel??

**Comment by S-Ali-Zaidi** [comment]
wondering the same!

**Comment by K0HYD** [comment]
Would sure love to see this. A detailed compile instruction sheet would work also.

**Reply by jdcrunchman** [reply]
I tried to DL the intel version,  but I have an M1 Mac,   can I use the Intel version on my Mac M1?

**Reply by AlexandreRouma** [reply]
It will work but performance will be reduced. Now that GitHub has added free M1 CI runners, there will soon be a M1 release.
